app/assets/javascripts/users.js.erb

```diff
@@ -50,6 +40,37 @@ function initMap() {
   }
 }
 
+function displayTargets() {
+  $.get('/user_targets').done(function(data) {
+    $.each(data.user_targets, function(i, val) {
+      var targetPos = {
+        lat: val['lat'],
```

Comment:
Use dot notation to access the object properties. Here are some examples: https://github.com/airbnb/javascript#properties--dot